packages/db/src/sdk/Item.ts

```diff
@@ -0,0 +1,64 @@
+ 
+  import Items, {
+    ItemsTable,
+  ItemsTableColumns,
+  ColumnsDefault
+} from "../schemas/items";
+
+
+
+import { Service, ServiceArgs } from "../types";
+
+export interface CreateItemInput {
+  id?: number;
+  name: string;
+  color: string;
```

Comment:
I don't think this needs to be a necessary param. I think we should continue to support color being optional